tests/test_dictionary.py

```diff
@@ -88,46 +92,47 @@ def test_dictionary_description_fail(dict_path, interface, raises):
 
 
 @pytest.mark.parametrize(
-    "dictionary_class, dictionary_path",
+    ("dictionary_class", "dictionary_path"),
     [
         (CanopenDictionaryV2, f"{PATH_RESOURCE}canopen/test_dict_can.xdf"),
         (EthernetDictionaryV2, f"{PATH_RESOURCE}ethernet/test_dict_eth.xdf"),
     ],
 )
-@pytest.mark.no_connection
+@pytest.mark.no_connection()
 def test_dictionary_v2_image(dictionary_class, dictionary_path):
     dictionary = dictionary_class(dictionary_path)
     assert isinstance(dictionary.image, str)
 
 
 @pytest.mark.parametrize(
-    "dictionary_class, dictionary_path",
+    ("dictionary_class", "dictionary_path"),
     [
         (CanopenDictionaryV2, f"{PATH_RESOURCE}canopen/test_dict_can.xdf"),
         (EthernetDictionaryV2, f"{PATH_RESOURCE}ethernet/test_dict_eth.xdf"),
     ],
 )
-@pytest.mark.no_connection
+@pytest.mark.no_connection()
```

Comment:
I didn't knwon that can be done, i don't like it... In pytest oficial documentation fixtures are used without parenthesis

Reply:
[PT023](https://docs.astral.sh/ruff/rules/pytest-incorrect-mark-parentheses-style/#pytest-incorrect-mark-parentheses-style-pt023) We can undo these changes.

Reply:
Done